### ISSUE TITLE
Template Error Incorrectly Flagged for Imported Module via tsconfig/compilerOptions/paths

### ISSUE THREAD

**Issue by vicatcu** [angular/vscode-ng-language-service#665]
**Describe the bug**

I was following [this article](https://devdactic.com/ionic-multi-app-shared-library/), and got it to a working state. It's available for review here: https://github.com/vicatcu/devdacticMulti

You can clone that repo, then from the root folder run:
```
ng build academyLib
ionic serve --project appOne
```
And it launches and works without any issues. However, if you open the `projects/appOne/src/app/home/home.component.html` in VS Code you will see it incorrectly complains about the `academy-academyLib` element with the following error detail:
```
'academy-academyLib' is not a known element:
1. If 'academy-academyLib' is an Angular component, then verify that it is part of this module.
2. If 'academy-academyLib' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.ng(0)
```

**Comment by ayazhafiz** [comment]
This seems to be a duplicate of #645.

**Comment by kyliau** [comment]
This has been fixed by the new Ivy-native language service, released in [v11.1.0](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0).
It's an opt-in feature for now, please give it a try and let us know if you have any feedback.
For the best editor experience, please make sure your project has [`strictTemplates`](https://angular.io/guide/angular-compiler-options#stricttemplates) enabled in `angularCompilerOptions`.
In case you run into similar bug, please file a new issue. I'll close this for now.
